app/models/concerns/clone.rb

```diff
@@ -6,10 +6,14 @@ module Clone
 
   def clone(params: {}, clean_formulas: true, position: nil, hidden: false)
     emails = params[:emails]&.map(&:downcase)
+    current_user_id = params[:user_id]
 
     if emails.blank?
-      return "Clone::#{de_constantize_modulize_name.classify}".
-        safe_constantize.new(self, clean_formulas: clean_formulas, position: position, params: params, hidden: hidden).execute
+      if current_user_id.present?
+        return clone_module.new(self, user_id: current_user_id, clean_formulas: clean_formulas, position: position, params: params, hidden: hidden).execute
+      end
```

Comment:
Additional "if" in this place will be confusing here. Can we move this logic to `Clone::Base` or another place? 🤔

Reply:
I don't see any changes

Reply:
Please check https://github.com/Michigan-State-University/cias-api/pull/135/commits/31e6ad15c81b8d418a8fbd83b1a7dffd6dc6eacf